src/responder/common/cache_req/plugins/cache_req_user_by_name.c

```diff
@@ -170,7 +170,7 @@ cache_req_user_by_name_dp_send(TALLOC_CTX *mem_ctx,
     errno_t ret;
 
     ret = cache_req_user_by_name_dpreq_params(mem_ctx, cr, result,
-                                             &string, &id, &flag);
+                                              &string, &id, &flag);
```

Comment:
I can see more indentation "violation" in this file and also on other places in cache_req.
e.g.
```
diff --git a/src/responder/common/cache_req/plugins/cache_req_group_by_name.c b/src/responder/common/cache_req/plugins/cache_req_group_by_name.c
index 3be0d5ea5..4b192f1a4 100644
--- a/src/responder/common/cache_req/plugins/cache_req_group_by_name.c
+++ b/src/responder/common/cache_req/plugins/cache_req_group_by_name.c
@@ -165,7 +165,7 @@ cache_req_group_by_name_dp_send(TALLOC_CTX *mem_ctx,
     errno_t ret;
 
     ret = cache_req_group_by_name_dpreq_params(mem_ctx, cr, result,
-                                             &string, &id, &flag);
+                                               &string, &id, &flag);
     if (ret != EOK) {
         return NULL;
     }
```

I would prefer to fix them at once and not in 30+ patches

Reply:
Well, I was not checking for indentation issues and didn't have the plan to do so. I just fixed the spot where I found it.

The same comment could be applied to "fixing typos" commit: https://github.com/SSSD/sssd/commit/1f2324abfbc69d95fe98a36216caa86a89cc6fb2.
Wouldn't there be better to fix all the typos at once as well? Not sure why the criteria changed between your patch and mine, but well ... not a new thing in this environment.

I'm updating the PR and dropping my patch.